cli/alert.go

```diff
@@ -6,17 +6,15 @@ import (
 	"fmt"
 	"net/http"
 	"net/url"
-	"path"
 	"strings"
 	"time"
 
+	"github.com/alecthomas/kingpin"
```

Comment:
Actually, no :) We need to use the v3-branch, since the pull request I made to kingpin to add support for config files is only present on that branch.